File: PRAK305/PRAK305-2310817110011-Ghani_Mudzakir.c
#include <stdio.h>

int main()
{
    int masukan_detik;
    scanf("%d", &masukan_detik);

    int hari = masukan_detik/86400;
    int sisa_detik_habis_hari = masukan_detik%86400;
    int jam = masukan_detik/3600;
    int habis_jam = masukan_detik%3600;
    int menit = habis_jam/60;
    int detik = habis_jam%60;

    if (hari == 0){
        printf("%02d : %02d : %02d\n", jam,menit,detik);
    }
    else if( hari != 0){
        printf("%d hari %02d : %02d : %02d\n", hari,jam,menit,detik);
    }

    return 0;
}

File: PRAK305/PRAK305-2310817110011-Ghani_Mudzakir.py
masukan_detik = int(input())

hari = int(masukan_detik/86400)
sisa_detik_habis_hari = int(masukan_detik%86400)
jam = int(sisa_detik_habis_hari/3600)
sisa_detik_habis_jam = int(masukan_detik%3600)
menit = int(sisa_detik_habis_jam/60)
detik = int(sisa_detik_habis_jam%60)

if hari == 0:
    print(f"{jam:02d}:{menit:02d}:{detik:02d}")

elif hari != 0:
    print(f"{hari:2d} hari {jam:02d}:{menit:02d}:{detik:02d}")




